exporter/tanzuobservabilityexporter/config.go

```diff
@@ -22,7 +22,7 @@ import (
 	"go.opentelemetry.io/collector/config/confighttp"
 )
 
-type TracesConfig struct {
+type tracesConfig struct {
```

Comment:
For config we encourage to be public to allow programmatically changes.

Reply:
Fixed!